ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/behavior/tree/RDXBehaviorTreeNodeCreationMenu.java

```diff
@@ -27,17 +27,23 @@
 
 public class RDXBehaviorTreeNodeCreationMenu
 {
-   private final RDXBehaviorTree tree;
-   private final BehaviorTreeTopologyOperationQueue topologyOperationQueue;
+   private final RDXBehaviorTree behaviorTree;
+   private final BehaviorTreeTopologyOperationQueue<RDXBehaviorTreeNode<?, ?>> topologyOperationQueue;
```

Comment:
Parameterizing this class allowed to make more things compile errors instead of runtime errors, and extract more common logic